### DISCUSSION TITLE
How to run moon on all targets ignoring touched files?

### DISCUSSION THREAD

**Discussion by rhuanbarreto** [moonrepo/moon#1256] [Q&A] [answered]
I want to run `moon ci` for all targets. Today it only runs over the touched files. But I want to run it over all projects regardless of touched files. How do I do this?

**Comment by milesj** [comment, marked as answer]
@rhuanbarreto You can't with `moon ci` but you can with `moon run`.

**Reply by rhuanbarreto** [reply]
Although the right would be to do a moon check --all

**Reply by milesj** [reply]
Ah right, whoops.
